### PR TITLE
BAH-2185-feature - [ Rohit ] - Configure the template for printing the visit details from the patient dashboard

### PR DESCRIPTION
**Pull request by rohit-yawalkar**
This PR is about:
 - Configuring the template for printing the visit details from the patient dashboard
 - Jira: https://bahmni.atlassian.net/browse/BAH-2185

### CODE REVIEW

File: ui/app/clinical/common/views/patientInfoPrint.html [hunk 1]
```diff
@@ -2,17 +2,19 @@
...
+            <td>Age: <span class="value">
+                {{patient.birthdate | dateToAge:visit.dateStarted}}
+                ({{patient.genderText.substring(6, patient.genderText.length - 7)}})
+            </span></td>
+            <td>Registration No: <span class="value">{{patient.identifier}}</span></td>
         </tr>
         <tr>
             <td>Village / Town: <span class="value">{{patient.address.cityVillage}}</span></td>
-            <td>P.O. <span class="value">{{patient.address.address2}}</span></td>
+            <td>P.O.: <span class="value">{{patient.address.address2}}</span></td>
```
Comment (angshu) [inline ui/app/clinical/common/views/patientInfoPrint.html:13]: remove . after P.O. .. as you already introducing :

File: ui/app/clinical/common/views/visitSummary.html [hunk 2]
```diff
@@ -28,65 +25,79 @@
...
+                                                section-title="::section | titleTranslate"
+                                                class="section-title"></bahmni-observation>
+                        </ul>
+                    </section>
+                </div>
+            </div>
+            <div ng-switch-when="diagnosis" class="section">
+                <section ng-if="::section.config" class="block dashboard-section section-title has-link">
+                    <bahmni-diagnosis patient-uuid="::patient.uuid" config="::section" show-latest-diagnosis="false"
+                                      show-ruled-out-diagnoses="::section.showRuledOutDiagnoses"></bahmni-diagnosis>
```
Comment (angshu) [inline ui/app/clinical/common/views/visitSummary.html:55]: is there a requirement for showing ruled out diagnosis?

File: ui/app/clinical/common/views/visitSummary.html [hunk 3]
```diff
@@ -28,65 +25,79 @@
...
                 </div>
             </div>
-            <div data-ng-switch-when="investigationResult">
+            <div data-ng-switch-when="labOrders" class="section">
                 <div ng-if="::section.config">
                     <investigation-results
                             params="::section.config"></investigation-results>
                 </div>
             </div>
-            <div data-ng-switch-when="radiology">
```
Comment (angshu) [inline ui/app/clinical/common/views/visitSummary.html:50]: any reason for removing radiology or pacsorder?

File: ui/app/clinical/common/views/visitTabPrint.html [hunk 4]
```diff
@@ -7,8 +7,8 @@
         <div class="print-header">
             <h1>
                 <img ng-if="visitTabConfig.getPrintConfigForCurrentTab().logo" class="logo" ng-src="{{visitTabConfig.getPrintConfigForCurrentTab().logo}}" alt="logo">
-                {{visitTabConfig.getPrintConfigForCurrentTab().title}}</h1>
-            <h2>{{visitTabConfig.getPrintConfigForCurrentTab().header}}</h2>
```
Comment (angshu) [inline ui/app/clinical/common/views/visitTabPrint.html:11]: why remove this?

File: ui/app/clinical/displaycontrols/investigationresults/views/investigationChart.html [hunk 5]
```diff
@@ -50,53 +52,5 @@ <h2 ng-click="toggleChart()" class="section-title row-click">
                 </tbody>
             </table>
         </div>
-        <div class="visible-print">
```
Comment (angshu) [inline ui/app/clinical/displaycontrols/investigationresults/views/investigationChart.html:53]: why remove this?

File: ui/app/common/patient/filters/age.js [hunk 6]
```diff
@@ -4,11 +4,11 @@ angular.module('bahmni.common.patient')
 .filter('age', function () {
     return function (age) {
         if (age.years) {
-            return age.years + " y";
+            return age.years + " year" + (age.years > 1 ? "s" : "");
```
Comment (angshu) [inline ui/app/common/patient/filters/age.js:7]: this would not be i18n. not that it as before .. but now its year, month, day - any specific reason?

Comment (angshu) [reply ui/app/common/patient/filters/age.js:7]: also why complicate with plural (s) ..

File: ui/app/clinical/common/views/visitSummary.html (hunk 2 above)
Comment (gsluthra) [reply ui/app/clinical/common/views/visitSummary.html:55]: Not really. Just need to print Confirmed/Suspected diagnosis in print out.

File: ui/app/clinical/common/views/visitSummary.html (hunk 3 above)
Comment (gsluthra) [reply ui/app/clinical/common/views/visitSummary.html:50]: Bahmni LITE doens't have pacs/radiology order configured. But in hindsight, realised we might need it. Can add it back when we add Radiology orders story. Just configuration in visit.json.

File: ui/app/clinical/common/views/visitTabPrint.html (hunk 4 above)
Comment (gsluthra) [reply ui/app/clinical/common/views/visitTabPrint.html:11]: We can print Tab Title. In config we should set title to "Patient Visit Summary".

File: ui/app/common/patient/filters/age.js (hunk 6 above)
Comment (gsluthra) [reply ui/app/common/patient/filters/age.js:7]: Yes.. agree..should use an i18n template string: `%age_in_years (%sex)`

File: ui/app/clinical/common/views/patientInfoPrint.html (hunk 1 above)
Comment (rohit-yawalkar) [reply ui/app/clinical/common/views/patientInfoPrint.html]: @angshu resolved this issue

File: ui/app/clinical/common/views/visitTabPrint.html (hunk 4 above)
Comment (rohit-yawalkar) [reply ui/app/clinical/common/views/visitTabPrint.html:11]: @gsluthra @angshu Added title as "Patient Visit Summary"

File: ui/app/common/patient/filters/age.js (hunk 6 above)
Comment (rohit-yawalkar) [reply ui/app/common/patient/filters/age.js]: @angshu @gsluthra Added the translations

**Comment by gsluthra** [comment]
Code is OK from my side.

**Pull request merged**